draft-irtf-cfrg-vdaf.md

```diff
@@ -217,48 +220,43 @@ fraction of people have experienced a given disease.  Thus systems that provide
-differentially private ensures that the degree to which any individual
-measurement influences the value of the aggregate result can be precisely
-controlled. For example, in systems like RAPPOR {{EPK14}}, each user samples
-noise from a well-known distribution and adds it to their measurement before
-submitting to the aggregation server. The aggregation server then adds up the
-noisy measurements, and because it knows the distribution from whence the noise
-was sampled, it can estimate the true sum with reasonable precision.
-
-Differentially private systems like RAPPOR are easy to deploy and provide a
-useful guarantee. On its own, however, DP falls short of the strongest privacy
-property one could hope for. Specifically, depending on the "amount" of noise a
-client adds to its measurement, it may be possible for a curious aggregator to
-make a reasonable guess of the measurement's true value. Indeed, the more noise
-the clients add, the less reliable will be the server's estimate of the output.
-Thus systems employing DP techniques alone must strike a delicate balance
+This problem is often formulated in terms of differential privacy (DP)
+{{Dwo06}}. Roughly speaking, a data aggregation system that is differentially
+private ensures that the degree to which any individual measurement influences
+the value of the aggregate result can be precisely controlled. For example, in
+systems like RAPPOR {{EPK14}}, each user samples noise from a well-known
+distribution and adds it to their measurement before submitting to the
+aggregation server. The aggregation server then adds up the noisy measurements,
+and because it knows the distribution from which the noise was sampled, it can
+estimate the true sum with reasonable accuracy.
+
+However, even when noise is added to the measurements, collecting them in the
+clear still reveals a significant amount of information to the collector. On
+the one hand, depending on the "amount" of noise a client adds to its
+measurement, it may be possible for a curious collector to make a reasonable
+guess of the measurement's true value. On the other hand, the more noise the
+clients add, the less reliable will be the server's estimate of the output.
+Thus systems relying solely on a DP mechanism must strike a delicate balance
 between privacy and utility.
 
 The ideal goal for a privacy-preserving measurement system is that of secure
 multi-party computation (MPC): No participant in the protocol should learn
 anything about an individual measurement beyond what it can deduce from the
-aggregate. In this document, we describe Verifiable Distributed Aggregation
-Functions (VDAFs) as a general class of protocols that achieve this goal.
+differentially private aggregate {{MPRV09}}. In this document, we describe
+Verifiable Distributed Aggregation Functions (VDAFs) as a general class of
+delegated MPC protocols that can be used to achieve this goal.
```

Comment:
What does "delegated" MPC mean? Is the distinction useful in this document?

Reply:
"Delegated" means a small set of servers carry out the computation on behalf of a large number of clients providing the inputs.

Reply:
Is that a term of art?  It's the first I've heard it.